iml-agent/src/cli.rs

```diff
@@ -233,6 +234,36 @@ pub enum StratagemClientCommand {
         #[structopt(flatten)]
         fidopts: FidInput,
     },
+
+    #[structopt(name = "filesync")]
+    /// Run FileSync action
+    FileSync {
+        #[structopt()]
+        /// push or pull
+        action: action_filesync::ActionType,
+
+        #[structopt(short = "r", long = "remote")]
+        /// remote fs path
+        target_fs: String,
+
+        #[structopt(parse(from_os_str))]
+        files: Vec<PathBuf>,
+    },
+
+    #[structopt(name = "cloudsync")]
+    /// Run CloudSync action
+    CloudSync {
+        #[structopt()]
```

Comment:
(same a above)

Reply:
It is, but this can be cleaned up in a follow-on.